docs/reference/crud.txt

```diff
@@ -863,6 +863,12 @@ the database up to the time it is saved. Any persistence operation clears the ch
    # Get the previous value for a field.
    person.name_was # "Alan Parsons"
 
+.. note::
+
+  Setting the associations on a document does not cause the ``changed`` or
+  ``changed_attributes`` hashes to be modified. This is true for all associations
+  whether referenced or embedded.
```

Comment:
FYI--In my app, we use a gem hack https://github.com/tablecheck/mongoid_relations_dirty_tracking that allows us to track this. For **embedded** (which are expected to behave like fields), there is a strong argument to add change tracking to Mongoid core in the future. (Not needed for referenced.)

Reply:
I agree, would you mind creating a ticket for this?

Reply:
I will do so, thanks!

Reply:
https://jira.mongodb.org/browse/MONGOID-5272